test/promises_aplus_adapter.js

```diff
@@ -13,11 +14,9 @@ require( "jsdom" ).env( "", function ( errors, window ) {
 		var deferred = jQuery.Deferred();
 
 		return {
-			get promise() {
-				return deferred.promise();
-			},
+			promise: deferred.promise(),
```

Comment:
I mainly changed it here since I was touching this file as well and having a regular property here means we might be able to run at least a subset of tests in IE8 (not relevant for this branch but still).